src/unix/fontutil.cpp

```diff
@@ -219,9 +215,9 @@ wxFontEncoding wxNativeFontInfo::GetEncoding() const
     return wxFONTENCODING_SYSTEM;
 }
 
-void wxNativeFontInfo::SetPointSize(float pointsize)
+void wxNativeFontInfo::SetFractionalPointSize(float pointsize)
 {
-    pango_font_description_set_size( description, pointsize * PANGO_SCALE );
+    pango_font_description_set_size( description, static_cast<gint>(pointsize * PANGO_SCALE) );
```

Comment:
Not tested, but would some king of _round_ be useful here instead of the cast to `gint`?